jbmc/regression/jbmc-json-ui/pointer-simplification/test.desc

```diff
@@ -1,6 +1,6 @@
 CORE
 test
-'.[] | select(has("result")) | .result | .[] | select(has("trace")) | .trace | .[] | select(has("value")) | .value | select(.name == "pointer") | {has_data: has("data")}' --json-ui --trace
+'.[] | select(has('result')) | .result | .[] | select(has('trace')) | .trace | .[] | select(has('value')) | .value | select(.name == 'pointer') | {has_data: has('data')}' --json-ui --trace
```

Comment:
This seems to be tripping up all the UNIX builds now (ubuntu and osx). 

Is there a way perhaps to have two of these commands, and dispatch based on OS detection at test runtime? I'm not sure we have one, but maybe it wouldn't be hard to come up with one?

If that's not possible/convenient for the time being, maybe this change could be reverted and the test marked as `KNOWNBUG` on windows?

Reply:
Thank you for digging into this one, I was quite confused.